### DISCUSSION TITLE
Questions about export the information of seed neighbors after tessellation

### DISCUSSION THREAD

**Discussion by diuky** [neperfepx/neper#636] [Q&A] [answered]
Hi All,

I'm a new user for neper and I'm very interested in various algorithms used in neper for tessellation. Now I am trying to modify the neut_tess_fprintf_cell function to output the information of the seed neighbors. However, after I modified the code and rebuilt it, I still get the same output file as the unmodified situation. I have even tried to only add a very simple fprintf line in the source, but still no change is detected. Could you help me with this? Thanks a lot!

**Comment by rquey** [comment, marked as answer]
I think that your `neper` command refers to the version installed on the system, not the local binary. Use `./neper` for the local binary.

This being said, also have a look at the `-statseed` option to get seed information.

**Reply by diuky** [reply]
Thanks so much! Now, the problem is solved.
